mineinabyss-features/src/main/kotlin/com/mineinabyss/helpers/HelperFunctions.kt

```diff
@@ -121,6 +73,9 @@ fun handleCurse(player: Player, from: Location, to: Location) {
     }
 }
 
+val Player.mount: LivingEntity?
+    get() = (vehicle ?: ModelEngineAPI.getMountManager()?.getMountedPair(uniqueId)?.base?.original) as? LivingEntity
```

Comment:
Dont like this extension since it says nothing about going through modelengine. Rename or put in a class related to modelengine util

Reply:
well it gets all mounts, so if ur riding a pig or modelengine mob, not purely modelengine aimed

Reply:
It's fine but I still think it leads to confusion as to why there's a mount and vehicle function and what the difference is. This wouldnt be a problem if this was just a function in a singleton because we'd know it's related to whatever that class is named, I think you need to be extremely specific and careful with top level extensions or just not make them top level extensions.